.github/workflows/test-rustflags-export.yml

```
@@ -0,0 +1,82 @@
+name: Test rustflags export
+# Exercises the rustflags inputs through the composite-action boundary so the
+# GITHUB_ENV heredoc the export step writes is parsed by a real runner and the
+# resulting RUSTFLAGS is observed by a later step.
+#
+# Driven on the release event because the nested setup-rust skips sccache for
+# releases, and the sccache post-step is unreliable under act.
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)